File: MPI/mpicheck/tests/functions/3.0/MPI_Improbe.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
       LOGICAL var_3
       INTEGER var_4
       INTEGER var_5(10)
       INTEGER var_6
        call mpi_improbe(var_0, var_1, var_2, var_3, var_4, var_5, var_6)
       call pmpi_improbe(var_0, var_1, var_2, var_3, var_4, var_5, var_6)
        end program main
    

File: MPI/mpicheck/tests/functions/3.0/MPI_Improbe.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    int var_0;
    int var_1;
    MPI_Comm var_2;
    int *var_3;
    MPI_Message *var_4;
    MPI_Status *var_5;
    int ret;
    /* calls */
    ret = MPI_Improbe(var_0, var_1, var_2, var_3, var_4, var_5);
    ret = PMPI_Improbe(var_0, var_1, var_2, var_3, var_4, var_5);
    return 0;
}
